index.html

```diff
@@ -4,63 +4,219 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Figma Template 1</title>
-    <link rel="stylesheet"
-          href="https://fonts.googleapis.com/css2?family=Crimson+Pro">
-    <link rel="stylesheet" href="style.css">   
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Crimson+Pro">
+    <link rel="stylesheet" href="style.css">
+    <title>Leon's Portfolio</title>
 </head>
 <body>
-    <header>
-        <a class="logo">Leon Espitia</a>
-        <nav class="menu_icon">
-            <div>
-              <img src="./assets/icons/Hamburguer.svg" alt=""/>
-            </div>  
-        </nav>
-    </header>
-
     <main>
```

Comment:
- [ ] You should use `Grid` layout to layout the project cards and `FlexBox` to layout the content of one project card.
![image](https://user-images.githubusercontent.com/44057863/170818736-b642bf9f-5e8f-4a18-bb8c-1b2cd8114fcc.png)